lib/kasket/visitor.rb

```diff
@@ -125,7 +125,7 @@ def visit_Array(node, *_)
     end
 
     def visit_Arel_Nodes_Casted(node, *_)
-      quoted(node.val)
+      quoted(node.val) if node.val
```

Comment:
I'm not 100% sure if we care, but I figure if you are trying to patch `nil` I think it would be safer to be explicit